src/main/java/com/avides/springboot/testcontainer/common/container/AbstractEmbeddedContainer.java

```diff
@@ -50,6 +50,8 @@ private static String getDockerHost()
         {
             Builder builder = DefaultDockerClientConfig.createDefaultConfigBuilder();
             Field declaredField = builder.getClass().getDeclaredField("dockerHost");
+            declaredField = builder.getClass().getDeclaredField("dockerHost");
+            declaredField = builder.getClass().getDeclaredField("dockerHost");
             declaredField.setAccessible(true);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make sure that this accessibility update is safe here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.avidesmedia.de/coding_rules#rule_key=squid%3AS3011)